app/src/main/java/org/fossasia/openevent/app/data/db/DatabaseRepository.java

```diff
@@ -0,0 +1,36 @@
+package org.fossasia.openevent.app.data.db;
+
+import com.raizlabs.android.dbflow.rx2.language.RXSQLite;
+import com.raizlabs.android.dbflow.sql.language.SQLOperator;
+import com.raizlabs.android.dbflow.sql.language.SQLite;
+import com.raizlabs.android.dbflow.structure.BaseModel;
+
+import org.fossasia.openevent.app.data.db.contract.IDatabaseRepository;
+
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+
+public class DatabaseRepository implements IDatabaseRepository {
```

Comment:
How are **updates** handled? 
For example - when user requests reload, how its gonna update the stored database?